src/hash_probe_from_file.cpp

```diff
@@ -24,9 +23,9 @@ void rpkg_function::hash_probe_from_file(std::string& input_path, std::string& f
         LOG_AND_EXIT("Error: The folder " + input_rpkg_folder_path + " to search for RPKG files for hash probe mode does not exist.");
     }
 
-    std::string H1 = "Z:\\HITMAN1\\";
-    std::string H2 = R"(Z:\HITMAN2\Runtime\)";
-    std::string H3 = R"(C:\Program Files\Epic Games\HITMAN3\Runtime\)";
+    const std::string H1 = R"(Z:\HITMAN1\)";
```

Comment:
This should just be removed along side importing them from these folders.

Reply:
cc @Notexe